v03_pipeline/lib/annotations/snv_indel_test.py

```diff
@@ -25,33 +25,37 @@ def test_allele_count_annotations(self) -> None:
             [
                 {
                     'id': 0,
-                    'ref_samples': hl.Struct(project_1={'a', 'c'}, project_2=set()),
-                    'het_samples': hl.Struct(project_1={'b', 'd'}, project_2=set()),
-                    'hom_samples': hl.Struct(project_1={'e', 'f'}, project_2=set()),
+                    'ref_samples': hl.Struct(project_1={'a', 'c'}, project_2=set(), R0607_gregor_training_project_=set()),
+                    'het_samples': hl.Struct(project_1={'b', 'd'}, project_2=set(), R0607_gregor_training_project_=set()),
+                    'hom_samples': hl.Struct(project_1={'e', 'f'}, project_2=set(), R0607_gregor_training_project_={'l', 'm'}),
```

Comment:
I'm a bit confused as to how this test is working... shouldn;t we be testing that the samples are NOT added?

Reply:
Ah.  So I wrote this so that the projects are added to the lookup table, and we just ignore them at gt_stats lookup time.  This means we're doing the unnecessary work of touching the lookup table.  

I went this route because it was conceptually simpler and easier to unit test, and I didn't think about trying to optimize away the lookup table work.  The code to skip adding to the lookup table itself isn't much harder to write though.

Reply:
no this is fine. we are going to rearchitect this whole table anyways, and when we rebuild it we can figure out what the better long term approach is

Reply:
yeah, that was my thought as well!